src/app/route/route-edit/route-edit.component.html

```diff
@@ -1,84 +1,59 @@
-<div class="container">
+<div class="container" style="padding-top: 10px;">
   <div class="row">
     <div class="col-xs-12">
-      <div class="row" *ngIf="currentImageUrl">
-        <div class="col-xs-12">
-          <div class="form-group text-center img-responsive">
-            <img [src]="currentImageUrl" alt="Current Route Image" style="max-width: 80%; max-height: 40vh; margin-bottom: 20px;">
-          </div>
-        </div>
-      </div>
-      <form [formGroup]="routeForm" (ngSubmit)="onSubmit()" enctype="multipart/form-data">
-        <div class="row">
-          <div class="col-xs-12" *ngIf="editMode">
-            <h2>ID: {{ id }}</h2>
-            <h4>User: {{ userLogin }}</h4>
-          </div>
-          <div class="col-xs-12" *ngIf="!editMode">
-            <h2>Adding new Route</h2>
-          </div>
+      <div class="card mb-4">
+        <div class="card-header">
+          <h2 *ngIf="!editMode" class="card-title">Adding New Route</h2>
+          <h2 *ngIf="editMode" class="card-title">Editing Route</h2>
         </div>
-        <div class="row">
-          <div class="col-xs-12">
-            <button
-              type="submit"
-              class="btn btn-success"
-              [disabled]="routeForm.invalid">Save
-            </button>
-            <button
-              type="button"
-              class="btn btn-danger"
-              (click)="goBack()">Cancel
-            </button>
-            <button
-              *ngIf="editMode"
-              type="button"
-              class="btn btn-danger"
-              (click)="onDelete()">Delete
-            </button>
-          </div>
-        </div>
-        <div class="row">
-          <div class="col-xs-12">
-            <div class="form-group">
-              <label for="name">Name</label>
-              <input
-                type="text"
-                id="name"
-                formControlName="name"
-                class="form-control">
+        <form (ngSubmit)="onSubmit()" [formGroup]="routeForm">
+          <div class="card-body">
+            <div class="container">
+              <div class="col-12 col-md-6 offset-md-3 text-center">
+                <div class="mb-3">
+                  <img *ngIf="!selectedFile && currentImageUrl" [src]="currentImageUrl" alt="Current Route Image"
+                       class="img-fluid w-100">
+                  <img *ngIf="selectedFile" [src]="imagePreview" alt="Current Route Image" class="img-fluid w-100">
+                  <img *ngIf="!currentImageUrl" alt="Current Route Image"
+                       class="img-fluid w-100 mb-3" src="https://loyaltystarter.io/wp-content/themes/unbound/images/No-Image-Found-400x264.png">
```

Comment:
![image](https://github.com/user-attachments/assets/58892b62-95d2-43fd-86d1-7d0ce2cc4b4f)

Reply:
bug to fix. 
compare with the map-location-edit:


```
<div class="mb-3">
                  <img *ngIf="!selectedFile && currentImageUrl" [src]="currentImageUrl" alt="Current Route Image"
                       class="img-fluid w-100">
                  <img *ngIf="selectedFile" [src]="imagePreview" alt="Current Route Image" class="img-fluid w-100">
                  <img *ngIf="!selectedFile && !currentImageUrl" alt="Current Route Image" class="img-fluid w-100"
                       src="https://loyaltystarter.io/wp-content/themes/unbound/images/No-Image-Found-400x264.png">

                </div>
```